Automattic/wp-calypso
# Task: Updated trial Plans page with Woo Express info

~Depends on https://github.com/Automattic/wp-calypso/pull/74140~

## Proposed Changes

* Update the Plans page for trial users to show WX plan information instead of Commerce.

## Testing Instructions

* Open the plans page (`plans/<site-slug>`) for a site under the Trial plan.
* Enable `plans/wooexpress-medium` flag
* Check that the buttons now add the Woo Express plan to the cart (instead of Commerce plan)
* Check that the information shown is about the Woo Express Medium (Performance) Plan

![image](https://user-images.githubusercontent.com/3801502/223549225-9777e229-9ed1-4cf6-a720-5c39830e27c6.png)
